tests/Keboola/DbExtractorConfig/ConfigTest.php

```diff
@@ -13,111 +15,228 @@ class ConfigTest extends AbstractConfigTest
-        $this->expectExceptionMessage($exceptionMessage);
+        $config = new Config($configurationArray, new ConfigDefinition());
+
+        $this->assertEquals($configurationArray, $config->getData());
+    }
+
+    public function testConfigRow(): void
+    {
+        $configurationArray = [
+            'parameters' => [
+                'outputTable' => 'in.c-main.auto-increment-timestamp',
+                'incremental' => true,
+                'data_dir' => '/code/tests/Keboola/DbExtractor/../../data',
+                'extractor_class' => 'MySQL',
+                'table' => [
+                    'tableName' => 'auto_increment_timestamp',
+                    'schema' => 'test',
+                ],
+                'name' => 'auto-increment-timestamp',
+                'incrementalFetchingColumn' => '_weird-I-d',
+                'primaryKey' => [],
+                'columns' => [],
+                'enabled' => true,
+            ],
+        ];
+
+        $config = new Config($configurationArray, new ConfigRowDefinition());
 
-        $Config = new Config(new ConfigRowDefinition());
-        $Config->validateParameters($config['parameters']);
+        $this->assertEquals($configurationArray, $config->getData());
+    }
+
+    public function testConfigActionRow(): void
+    {
+        $configurationArray = [
+            'parameters' => [
+                'data_dir' => '/code/tests/Keboola/DbExtractor/../../data',
+                'extractor_class' => 'MySQL',
+                'db' => [
+                    'host' => 'mysql',
+                    'user' => 'root',
+                    '#password' => 'rootpassword',
+                    'database' => 'test',
+                    'port' => 3306,
+                ],
+            ],
+        ];
+
+        $config = new Config($configurationArray, new ActionConfigRowDefinition());
+
+        $this->assertEquals($configurationArray, $config->getData());
     }
 
     public function testInvalidConfigQueryIncremental(): void
     {
-        $config = $this->getConfig(self::DRIVER);
-        unset($config['parameters']['tables']);
-        $config['parameters']['incrementalFetchingColumn'] = 'test';
-        $config['parameters']['query'] = 'select 1 from test';
-        $config['parameters']['outputTable'] = 'fake.output';
-
-        $exceptionMessage = 'Invalid configuration for path "parameters": ';
-        $exceptionMessage .= 'Incremental fetching is not supported for advanced queries.';
+        $configurationArray = [
+            'parameters' => [
+                'outputTable' => 'fake.output',
+                'data_dir' => '/code/tests/Keboola/DbExtractor/../../data',
+                'extractor_class' => 'MySQL',
+                'query' => 'select 1 from test',
+                'incrementalFetchingColumn' => 'test',
+            ],
+        ];
+
+        $exceptionMessage = 'Incremental fetching is not supported for advanced queries.';
```

Comment:
Tady je lepší tam dát celou tu message. Jinak nepoznáš, když tam později něco nečekaného přibyde...

Reply:
tohle je celá message - vyhazuje to:
https://github.com/keboola/db-extractor-config/blob/07c976e5f6acc2cf1293484c3c15171fd4a47817/src/Configuration/ConfigRowDefinition.php#L99

Reply:
Jo, jasně, sorry!